ci/prow/deployments/horologuim.yaml

```diff
@@ -0,0 +1,40 @@
+# Copyright 2019 The Knative Authors
```

Comment:
typo: this file should be named horologium.yaml

Reply:
good catch, done